docs/docs_generator.rb

```diff
@@ -3,10 +3,19 @@
 # This must be run from /docs directory
 
 require 'mustache'
+require 'fileutils'
 require 'puppet'
 
 $LOAD_PATH << '.'
 
+FOLDER_TO_NAME_MAP = {
+  'backup' => 'Managed Backup',
+  'user' => 'User Management',
+  'compute-engine' => 'Compute Engine',
+  'kubernetes' => 'Managed Kubernetes',
+  'dbaas-postgres' => 'DbaaS Postgres',
```

Comment:
Maybe it is better if we use `DBaaS` to be in sync with the official doc. What do you think?

Reply:
Done, thank you!